### DISCUSSION TITLE
Best way to respond to text messages

### DISCUSSION THREAD

**Discussion by ecatugy** [WhiskeySockets/Baileys#100] [General]
## Question: Text Message Status 

**Problem:** What's the best way to know if a text message has been answered?

**Context:** Previously, we had the option to use buttons for this task. But now, it's no longer viable.

**Proposed Solution:** I'm thinking about handling it with text. However, I'll need an ID from the message I send to respond appropriately in the `messages.upsert` event.

Does anyone have a code example for this?

Or perhaps, is there a more efficient solution? Any input is appreciated.

Thanks in advance.

**Comment by andresayac** [comment]
I recommend you use this as a guide
https://github.com/codigoencasa/bot-whatsapp

I made an abridged version to be able to create a bot however it is not so close to your answer, from here you can use the find as buttons as I read the update event to get the selected option.
https://github.com/andresayac/bot_baileys


I repeat I don't know how close it is to your answer but the idea is that you can have a guide on how to do it.
